docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
Sean has requested a legal review on the example modules, please wait until legal has confirmed that our overview is accurate.

Reply:
I've just requested the legal review 1 minute ago. It will probably take legal some time to answer. My last request took ~2 weeks.